Testsuite/test/post_process_ctest_results.py

```diff
@@ -3,12 +3,13 @@
 import re
 import os
 
-report_file=sys.argv[1]
-report_name=sys.argv[2]
-global_report_name=sys.argv[3]
-rx=re.compile('(.*Configuring (examples|demo|test)*( in )*(test\/|examples\/|demo\/)*)((?!done)\w+)')
-rx_demo=re.compile('.*in demo\/')
-rx_examples=re.compile('.*in examples\/')
+input_report_file=sys.argv[1]
```

Comment:
Better use `_name` as a suffix to `input_report_file`. Reserve `input_report_file` for the actual file descriptor once the file is open.